src/Controls/src/Build.Tasks/nuget/buildTransitive/netstandard2.0/Microsoft.Maui.Controls.SingleProject.targets

```diff
@@ -61,6 +61,65 @@
 
   </Target>
 
+  <!--
+    Workaround for the issue where the resource starts with the Resources/ prefix,
+    either in the Identity or Link metadata. The fix is to assume that was not intended
+    since that is how it worked in 8.0.60 and earlier.
+    However, if there is a LogicalName metadata, we will skip processing as that is what
+    was explicitly set (even though the build will fail).
+  -->
+  <PropertyGroup>
+    <CollectBundleResourcesDependsOn Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios' or $([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'maccatalyst'">
+      _MauiRemoveResourcesPrefixFromBundleResource;
+      $(CollectBundleResourcesDependsOn);
+    </CollectBundleResourcesDependsOn>
+  </PropertyGroup>
+  <Target Name="_MauiRemoveResourcesPrefixFromBundleResource">
+    <!-- Collect the items into 2 categories: Link or no Link -->
+    <ItemGroup>
+      <_MauiBundleResourceWithLogicalName
+        Include="@(BundleResource->HasMetadata('LogicalName'))" />
+      <_MauiBundleResourceWithLink
+        Include="@(BundleResource->HasMetadata('Link'))"
+        Exclude="@(_MauiBundleResourceWithLogicalName)" />
+      <_MauiBundleResourceWithoutLink
+        Include="@(BundleResource)"
+        Exclude="@(_MauiBundleResourceWithLink);@(_MauiBundleResourceWithLogicalName)" />
+    </ItemGroup>
+    <!-- Set the LogicalName to be a path relative to Resources -->
+    <ItemGroup>
+      <_MauiBundleResourceWithRelativeLogicalName
+        Include="@(_MauiBundleResourceWithLink)"
+        Condition="@(_MauiBundleResourceWithLink->Count()) > 0"
+        LogicalName="$([MSBuild]::MakeRelative('$(MSBuildProjectDirectory)/Resources/', '$(MSBuildProjectDirectory)/%(Link)'))" />
+      <_MauiBundleResourceWithRelativeLogicalName
+        Include="@(_MauiBundleResourceWithoutLink)"
+        Condition="@(_MauiBundleResourceWithoutLink->Count()) > 0"
+        LogicalName="$([MSBuild]::MakeRelative('$(MSBuildProjectDirectory)/Resources/', '%(FullPath)'))" />
```

Comment:
I think they normally still put quotes around:
```
Condition="'@(_MauiBundleResourceWithLink->Count())' > '0'"
```
In case either side evaluates to an empty string.